test/combined_dataset_test.py

```diff
@@ -59,8 +59,8 @@ def test_combined_county_has_some_data(fips):
     assert latest.get_record_for_fips(fips=fips)[CommonFields.DEATHS] > 1
 
 
-# Check some counties picked arbitrarily: (Marion County, IN)/18097 and (Harris County, TX)/48201
-@pytest.mark.parametrize("fips", ["18097", "48201"])
+# Check some counties picked arbitrarily: (Orange County, CA)/06059 and (Harris County, TX)/48201
+@pytest.mark.parametrize("fips", ["06059", "48201"])
```

Comment:
Had to change this since we're disabling the valorum positive_tests / negative_tests for Indiana.